cpp/src/arrow/adapters/orc/CMakeLists.txt

```diff
@@ -26,3 +26,22 @@ install(FILES adapter.h DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/arrow/adapters/
 configure_file(arrow-orc.pc.in "${CMAKE_CURRENT_BINARY_DIR}/arrow-orc.pc" @ONLY)
 install(FILES "${CMAKE_CURRENT_BINARY_DIR}/arrow-orc.pc"
         DESTINATION "${CMAKE_INSTALL_LIBDIR}/pkgconfig/")
+
+set(ORC_MIN_TEST_LIBS ${GTEST_MAIN_LIBRARY} ${GTEST_LIBRARY})
+
+if(ARROW_BUILD_STATIC)
```

Comment:
this doesn't seem to follow the pattern in other directories, where we seem to create both static and dynamic lib paths, but I'm not an expert on CMake.  @xhochy?

Reply:
Here I followed the pattern in src/parquet/CMakeLists.txt

Reply:
This is just setting the preferred libs for testing. This is fine.